bitnami/etcd/README.md

```diff
@@ -268,6 +270,17 @@ If `startFromSnapshot` is enabled at the same time than `disasterRecovery`, the
 
 > **Note**: Disaster recovery feature requires using volumes with ReadWriteMany access mode. For instance, you can use the stable/nfs-server-provisioner chart to provide NFS PVCs.
 
+### Auto Compaction
+
+Since etcd keeps an exact history of its keyspace, this history should be periodically compacted to avoid performance degradation and eventual storage space exhaustion. Compacting the keyspace history drops all information about keys superseded prior to a given keyspace revision. The space used by these keys then becomes available for additional writes to the keyspace.
+
+You can enable auto compaction by using following parameters:
+
+```console
+autoCompactionMode=periodic
+autoCompactionRetention=10m
+```
+
```

Comment:
```suggestion
### Auto Compaction

Since etcd keeps an exact history of its keyspace, this history should be periodically compacted to avoid performance degradation and eventual storage space exhaustion. Compacting the keyspace history drops all information about keys superseded prior to a given keyspace revision. The space used by these keys then becomes available for additional writes to the keyspace.

`autoCompactionMode`, by default periodic. Valid values: ‘periodic’, ‘revision’. 
- 'periodic' for duration based retention, defaulting to hours if no time unit is provided (e.g. ‘5m’). 
- 'revision' for revision number based retention.

`autoCompactionRetention` for mvcc key value store in hour, by default 0, means disabled

You can enable auto compaction by using following parameters:

```console
autoCompactionMode=periodic
autoCompactionRetention=10m
```

```